Tests/PowerSkillTests/CustomEntitySearchTests/TestData.cs

```diff
@@ -106,21 +105,11 @@ public static class TestData
ce"":2.0},{""category"":""customEntity"",""value"":""so"",""offset"":12,""confidence"":1.0},{""category"":""customEntity"",""value"":""on"",""offset"":54,""confidence"":2.0},{""category"":""customEntity"",""value"":""now"",""offset"":67,""confidence"":2.0},{""category"":""customEntity"",""value"":""the fix was"",""offset"":0,""confidence"":10.0},{""category"":""customEntity"",""value"":""fix"",""offset"":4,""confidence"":10.0},{""category"":""customEntity"",""value"":""was so"",""offset"":8,""confidence"":10.0},{""category"":""customEntity"",""value"":""so"",""offset"":12,""confidence"":10.0},{""category"":""customEntity"",""value"":""simple, I"",""offset"":15,""confidence"":10.0},{""category"":""customEntity"",""value"":""I overlooked it"",""offset"":23,""confidence"":7.0},{""category"":""customEntity"",""value"":""overlooked it"",""offset"":25,""confidence"":5.0},{""category"":""customEntity"",""value"":""it"",""offset"":36,""confidence"":10.0},{""category"":""customEntity"",""value"":""Should work"",""offset"":42,""confidence"":10.0},{""category"":""customEntity"",""value"":""work on all"",""offset"":49,""confidence"":10.0},{""category"":""customEntity"",""value"":""on all tests"",""offset"":54,""confidence"":10.0},{""category"":""customEntity"",""value"":""all tests"",""offset"":57,""confidence"":10.0},{""category"":""customEntity"",""value"":""tests now"",""offset"":61,""confidence"":10.0},{""category"":""customEntity"",""value"":""now"",""offset"":67,""confidence"":10.0},{""category"":""customEntity"",""value"":""the fix"",""offset"":0,""confidence"":9.0},{""category"":""customEntity"",""value"":""was so"",""offset"":8,""confidence"":9.0},{""category"":""customEntity"",""value"":""so simple"",""offset"":12,""confidence"":9.0},{""category"":""customEntity"",""value"":""simple, I"",""offset"":15,""confidence"":9.0},{""category"":""customEntity"",""value"":""I overlooked it"",""offset"":23,""confidence"":5.0},{""category"":""customEntity"",""value"":""overlooked it"",""offset"":25,""confidence"":3.0},{""category"":""customEntity"",""value"":""Should"",""offset"":42,""confidence"":8.0},{""category"":""customEntity"",""value"":""work on"",""offset"":49,""confidence"":8.0},{""category"":""customEntity"",""value"":""on all tests"",""offset"":54,""confidence"":9.0},{""category"":""customEntity"",""value"":""all"",""offset"":57,""confidence"":9.0},{""category"":""customEntity"",""value"":""tests now"",""offset"":61,""confidence"":9.0},{""category"":""customEntity"",""value"":""now"",""offset"":67,""confidence"":9.0}],""entitiesFound"":[""fix"",""soo "",""overlooking"",""overlooked""]},""errors"":[],""warnings"":[{""message"":""The provided fuzzy offset of 10, is larger than the length of the provided word, \""fix\"".""},{""message"":""The provided fuzzy offset of 10, is larger than the length of the provided word, \""soo \"".""},{""message"":""The provided fuzzy offset of 10, is larger than the length of the provided word, \""overlooked\"".""}]}]}";
```

Comment:
Serialized JSON is harder to read and maintain. Can we define this as an anonymous object and serialize it in code instead?

Reply:
going to won't fix for this PR. I plan on taking another stab at refactoring tests in the next PR

---
In reply to: [330754632](https://github.com/Azure-Samples/azure-search-power-skills/pull/14#discussion_r330754632) [](ancestors = 330754632)